src/bin/nydus-image/main.rs

```diff
@@ -1321,7 +1321,10 @@ impl Command {
 
                 Some(Arc::new(local_fs))
             }
-            None => Self::get_backend(matches, "unpacker").ok(),
+            None => Some(BlobFactory::new_backend(
+                config.backend.as_ref().unwrap(),
```

Comment:
It's not safe to unpack, let's make sure that these options must be specified as one of them: `"backend-config"`, `"blob-dir"`, `"blob"`.

Reply:
Done.